src/hive.hpp

```diff
@@ -8,6 +8,11 @@ class Hive : public Location {
     public:
         Hive() : Hive(0, 0) { };
         Hive(int x, int y);
+        bool isPlant() override;
+        void depositNectar(float amount);
+        float getNectar();
+    private:
+        float nectar;
```

Comment:
Nectar value isn't set; I get a crazy big negative Nectar in Hive value when I run the simulation